tests/misc/test_access_links.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import subprocess
+import hashlib
+
+# This test is designed to verify the accessibility of the XOA deployment script
+#
+# Requirements:
+# - an XCP-ng host with xcp-ng-release >= 8.3.0.29
+
+@pytest.mark.parametrize("command_id", ["curl", "wget"])
+@pytest.mark.parametrize("url_id", [
+    "xoa",
+    "xcp-ng",
```

Comment:
`-` is a bit special, being used as separator between parameters when naming the test, compare:
```
tests/misc/test_access_links.py::test_access_links[xoa-wget]
tests/misc/test_access_links.py::test_access_links[xcp-ng-curl]
```

Let's avoid this when possible :)